### PR TITLE
Release notes for 1.19.x

### LINKED ISSUE #13648: Release Notes for 1.19.x Release #1 (Release Channel)
### Release Notes:

* Remove chrome's noscript UI #199
* Improve ability to recognize the difference between a private window and private window with Tor #2735
* Native IPFS support in Brave #10220
* [Desktop] brave://discards is blank #10451
* Sync - history page should show tabs from other devices #10473
* Brave Ads are not shown after restarting the browser if the catalog fails to download and a previous bundle exists #10904
* Desktop sync v2 does not sync all tabs from other computers #11183
* Use Greaselion for Twitch publishers #11466
* sometimes geolocation no longer working (uses old endpoints) #11862
* Change the directory path of admin policy on linux #12426
* Click on the `Open in Tor` link from Guest window doesn't connect to TOR network #12470
* Remove duplicate NTP controls from brave://settings/newTab #12505
* Update NTP background images for 2020/2021 #12508
* 5th background image (yellow) isn't showing up on tip banner #12596
* Replace "Add Card" tab with sub-tab "Edit Cards" #12625
* TOR not working with Multiple Profiles Active #12650
* Autoplay will have ask prompt if global content setting is toggled #12795
* Crash in ProxyConfigServiceTor #12798
* Number formatting incorrect for Crypto.com widget #12816
* Brave Ads crash when refilling unblinded tokens after downloading catalog due to invalid blinded token #12836
* Prevent auto downloading & opening a PDF file on click #12902
* Brave Today intro card should dismiss #12960
* Match redirect chain URLs against Brave Ad conversion URL patterns #13033
* Brave Today history scoring should not be limited to 100 items #13041
* [Desktop] Update Brave Rewards panel within welcome UI #13119
* Updated pre-populated search engine list #13283
* Strip referer to strict-origin-when-cross-origin in all cases #13464
* Upgrade from Chromium 88.0.4324.89 to Chromium 88.0.4324.96. #13637

### PR DESCRIPTION
**Pull request by kjozwiak**
Resolves #13648

## Submitter Checklist:

- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed).
- [ ] Requested a security/privacy review as needed.

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

N/A

### CODE REVIEW

File: CHANGELOG_DESKTOP.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+ - Fixed autoplay dialog being displayed under URL bar if global content settings under brave://settings/content/autoplay is toggled. ([#12795](https://github.com/brave/brave-browser/issues/12795))
+ - Fixed brave://discards loading blank page. ([#10451](https://github.com/brave/brave-browser/issues/10451))
+ - Fixed incorrect number formatting for Crypto.com widget. ([#12816](https://github.com/brave/brave-browser/issues/12816))
+ - Fixed Brave Today intro card not being dismissed after being viewed. ([#12960](https://github.com/brave/brave-browser/issues/12960))
+ - Fixed Brave Today history scoring being limited to 100 items. ([#13041](https://github.com/brave/brave-browser/issues/13041))
+ - Fixed geolocations using the wrong endpoints in certain cases. ([#11862](https://github.com/brave/brave-browser/issues/11862))
+ - Fixed PDF files automatically downloading and opening on click. ([#12902](https://github.com/brave/brave-browser/issues/12902))
+ - Upgraded Chromium to 88.0.4324.89. ([#13637](https://github.com/brave/brave-browser/issues/13637))
```
Comment (LaurenWags) [inline CHANGELOG_DESKTOP.md:32]: CR bump number needs to be updated to 88.0.4324.96

**Review by rebron — APPROVED** [review]
++ Looks good.

File: CHANGELOG_DESKTOP.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+ - Update referrer handling for better compatibility. ([#13464](https://github.com/brave/brave-browser/issues/13464))
+ - Updated directory path of admin policy on Linux. ([#12426](https://github.com/brave/brave-browser/issues/12426))
+ - Implemented ad conversion against the complete URL chain of trust. ([#13033](https://github.com/brave/brave-browser/issues/13033))
+ - Removed Chromium noscript dialog from the URL bar. ([#199](https://github.com/brave/brave-browser/issues/199))
+ - Removed duplicate controls from brave://settings/newTab. ([#12505](https://github.com/brave/brave-browser/issues/12505))
+ - Replaced the "Add Card" tab under New Tab Page with sub-tab "Edit Cards". ([#12625](https://github.com/brave/brave-browser/issues/12625))
+ - Fixed ad crash when refilling unblinded tokens after downloading catalog due to invalid blinded tokens. ([#12836](https://github.com/brave/brave-browser/issues/12836))
+ - Fixed ads not being displayed after restart if catalog fails to downlead but a previous catalog bundle exists. ([#10904](https://github.com/brave/brave-browser/issues/10904))
```
Comment (LaurenWags) [inline CHANGELOG_DESKTOP.md:19]: downlead --> download

**Review by LaurenWags — APPROVED** [review]
++ lgtm

**Review by rebron — APPROVED** [review]
++. Good eye @LaurenWags

**Pull request merged**